ssz/sedes/serializable.py

```diff
@@ -214,127 +253,103 @@ def _get_class_namespace(cls):
         yield from cls.__slots__
 
 
-class SerializableBase(abc.ABCMeta):
+class MetaSerializable(abc.ABCMeta):
 
-    def __new__(cls, name, bases, attrs):
-        super_new = super(SerializableBase, cls).__new__
+    def __new__(mcls, name, bases, namespace):
+        declares_fields = "fields" in namespace
 
-        serializable_bases = tuple(b for b in bases if isinstance(b, SerializableBase))
-        has_multiple_serializable_parents = len(serializable_bases) > 1
-        is_serializable_subclass = any(serializable_bases)
-        declares_fields = 'fields' in attrs
+        if declares_fields:
+            has_fields = True
+            fields = namespace.pop("fields")
+            try:
+                sedes = Container(fields)
+            except ValueError as exception:
```

Comment:
Since `ValueError` is a common standard library exception maybe the source of this should be converted to an `eth_utils.ValidationError` to ensure we're catching the right thing?  Also probably worth a comment explaining why this is being caught and re-raised.